project/Build.scala

```diff
@@ -1440,6 +1460,130 @@ object Build {
       settings(commonBenchmarkSettings).
       enablePlugins(JmhPlugin)
 
+    def asScala3doc: Project = {
```

Comment:
IMHO we should keep most of the settings in scala3doc/build.sbt, since  if we move them to top level all changes here will requrie to run full community build (and we may not want that).

Reply:
I don't think that's doable. The primary reason behind merging the two repos together is that we want their builds to be unified, so that Scala3doc can be easily compiled with latest compiler (which is practically necessary for quickly fixing compiler issues to help Scala3doc) and so that compiler sub-projects and community build projects can be easily documented with latest Scala3doc. I can't see any good way of doing that other than having Scala3doc be a sub-project here.